museek/data_element.py

```diff
@@ -85,6 +85,23 @@ def mean(
         if flags is None:
             return self._mean(axis=axis)
         return self._flagged_mean(axis=axis, flags=flags)
+    
```

Comment:
New methods should receive at least 1 new unit test each. In this case in `test/museek/test_data_element.py`. The new methods in this PR are very similar to existing ones, hence the new unittests can also be very similar to the tests of the existing methods.